Public/New-HTMLSection.ps1

```diff
@@ -18,6 +18,7 @@ Function New-HTMLSection {
         [string][ValidateSet('row', 'row-reverse', 'column', 'column-reverse')] $Direction,
         [string][ValidateSet('flex-start', 'flex-end', 'center', 'space-between', 'space-around', 'stretch')] $AlignContent,
         [string][ValidateSet('stretch', 'flex-start', 'flex-end', 'center', 'baseline')] $AlignItems
+        [string][ValidateSet('flex-start', 'flex-end', 'center')] $JustifyContent = 'flex-start' # Add $JustifyContent parameter and set default to flex-start
```

Comment:
Shouldn't we allow no choice at all so that when someone doesn't use it, it's skipped and doesn't impact view by accident? Forcing it to flex-start potentially has an impact.

Also not sure the comment is needed. It's pretty clear what it does by adding JustifyContent parameter :-) Since we have git we don't need to track who did what :-)

Reply:
Removed the set value and the in-line comment.